filer/management/commands/filer_check.py

```diff
@@ -112,3 +121,30 @@ def walk(prefix):
         filer_public = filer_settings.FILER_STORAGES['public']['main']
         storage = import_string(filer_public['ENGINE'])()
         walk(filer_public['UPLOAD_TO_PREFIX'])
+
+    def image_dimensions(self, options):
+        from tests.utils.custom_image.models import Image
+        from filer.models.imagemodels import Image
+        from django.db.models import Q
+        from easy_thumbnails.VIL import Image as VILImage
+        from filer.utils.compatibility import PILImage
+        import easy_thumbnails
+
+        no_dimensions = Image.objects.filter(
+            Q(_width=0) | Q(_width__isnull=True)
+        )
+        self.stdout.write(f"trying to set dimensions on {no_dimensions.count()} files")
+        for image in no_dimensions:
+            try:
+                imgfile = image.file.file
+            except ValueError:
+                imgfile = image.file_ptr.file
+            imgfile.seek(0)
+            if image.file.name.endswith('.svg'):
+                image._width, image._height = VILImage.load(imgfile).size
+            else:
+                pil_image = PILImage.open(imgfile)
+                self._width, self._height = pil_image.size
```

Comment:
```suggestion
                image._width, image._height = pil_image.size
```

I am not sure if `self` is correct here...